.gitattributes

```diff
@@ -0,0 +1 @@
+*.ts.snap linguist-generated=true
```

Comment:
this tells github to collapse the snapshot file diff (because it's an autogenerated file)